src/pages/cli/migration/lazy-load-custom-selection-set.mdx

```diff
@@ -0,0 +1,361 @@
+export const meta = {
```

Comment:
Overall feedback:
- Reference https://docs.amplify.aws/cli/graphql/data-modeling/#has-one-relationship for the writing style on relationships. "has-one" -> "Has One" or "@hasOne"
- "Amplify Swift" needs to be "Amplify for Swift"
- Don't use title casing for "Lazy Loading" or "Custom Selection Sets", rather "lazy loading", "custom selection sets"